src/constants.ts

```diff
@@ -3,4 +3,3 @@ import { parseEther } from 'viem';
 export const createRollupDefaultRetryablesFees = parseEther('0.125');
 
 export const createTokenBridgeDefaultRetryablesFees = parseEther('0.02');
-export const createTokenBridgeDefaultGasLimit = 5_000_000n;
```

Comment:
I decided on removing this default since gas limit will be different depending on if you're deploying an L2 or an L3 token bridge